### DISCUSSION TITLE
1,2 options

### DISCUSSION THREAD

**Discussion by midimusic** [GrandOrgue/grandorgue#1649] [General]
In the "Discussion" section:

Does anybody know what the 1,2 option box under the heading Discussions does?? It can select 1 or 2 but does not appear to do anything. These options also appear inside discussion posts but again do not appear to do anything.

Whilst on the subject of oddities - some discussions have an "Answered" statement. Who decides and how is this activated?

csw900

**Comment by ClarkeBrunt** [comment]
I can't spot a '1.2' option on the screen. But as for "Answered", clicking on the categories of discussion, 'Developer' and 'Help' say "Answers enabled", but 'General doesn't'. And there is a box on the screen (sometimes) saying "Please mark someone's comment as the answer if it answers your question". So my guess is the the original poster has the option to mark something as 'Answered' so long as it's in one of thowse categories.

Also: the discussions area seems to default to only showing the 'Unanswered' questions (in categories where available). I can change the filter to 'all', but is changes back to 'is:open' when I click on various things. Think I'd rather see all discussions by default, in case they are interesting.

It's gone very quiet in general in the discussions recently. Probably why we're now discussing the discussions themselves!

**Comment by midimusic** [comment]
@ClarkeBrunt 
Thanks for the comments. The 1,2 options are in a column under the heading "Discussions" - the options sometimes go up to 3. Your post has a 1,2 options box at the bottom left corner. As far as I can tell these options do absolutely nothing. I suspect nobody really knows what they are for.

I agree that "Answered" questions ought to be  always be shown unless specifically filtered out. However it is interesting to see what questions remain unanswered.

csw900

**Comment by ClarkeBrunt** [comment]
I see the '1,2' thing now.  I thought you meant that it said both numbers at the same time. It's about 'upvoting'.

[Here](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-discussion) is github documentation on it. Still not sure what good it does, or how/where upvoting something gives it more prominence. Anyway, I've upvoted the two comments by @midimusic (from 1 to 2), which I see I could take away by clicking again...

[Later edit:] OK - when viewing the list of discussions, Sort by 'top' etc. sorts by number of upvotes, and when viewing an individual discussion, 'Top' (as opposed to 'Oldest'/'Newest') sorts the comments. All very useful, I'm sure... Meanwhile: anyone want to talk about this 'GrandOrgue' thing?

**Comment by midimusic** [comment]
@ClarkeBrunt 
Thanks for your research. I would say it is a completely useless feature because anyone can change the upvotes any time they like. I also think that sorting by Top is useless because it gets the discussion out of order. Some complex discussions are hard enough to follow without get all the comments out of order.

csw900
